mgm/src/lib.rs

```diff
@@ -0,0 +1,271 @@
+//! Generic implementation of [Multilinear Galous Mode][1] [AEAD] construction.
+//!
+//! # Example
+//! ```
+//! # #[cfg(feature = "alloc")]
+//! # {
+//! use mgm::Mgm;
+//! use kuznyechik::Kuznyechik;
+//! use mgm::aead::{Aead, NewAead, generic_array::GenericArray};
+//!
+//! let key = GenericArray::from_slice(b"an example very very secret key.");
+//! let cipher = Mgm::<Kuznyechik>::new(key);;
```

Comment:
double `;;` at the end of the list

Reply:
Surprised `clippy` didn't catch that

Reply:
Currently `clippy` does not check code examples, see: rust-lang/rust#56232